examples/helloapache/README.md

```diff
@@ -1,8 +1,10 @@
+# helloapache
+
 This is an atomic application based on the nulecule specification. Kubernetes is currently the only supported provider. You'll need to run this from a workstation that has the atomic CLI and kubectl client that can connect to a kubernetes master.
 
-It's a single pod based on the centos/httpd image, but you can use your own.
+It's a single container application based on the centos/httpd image, but you can use your own.
 
```

Comment:
Fair point as we support multiple providers. I was inclined towards k8s as the Project Atomic ecosystem users k8s more than anything else at this point of time i.e. OpenShift,  Atomic Enterprise, Atomic hosts etc uses k8s internally.